common/scripts/ci/run_cypress_recording.sh

```diff
@@ -78,6 +78,8 @@ mkdir -p ./recordings/mappings/$SDK_BACKEND
 docker build --no-cache --file Dockerfile_local -t $IMAGE_ID . || exit 1
 
 echo "⭐️ 6/8 Run isolated recording against TEST_BACKEND=$TEST_BACKEND."
+export USER_UID=$(id -u $USER)
```

Comment:
Fix different user inside and outside container

```
13:33:51 ⭐️ 6/8 Run isolated recording against TEST_BACKEND
13:33:51 time="2023-11-08T07:33:51+01:00" level=warning msg="The \"USER_UID\" variable is not set. Defaulting to a blank string."
13:33:51 time="2023-11-08T07:33:51+01:00" level=warning msg="The \"USER_GID\" variable is not set. Defaulting to a blank string."
13:33:51 time="2023-11-08T07:33:51+01:00" level=warning msg="The \"DOCKER_BUILD_URL\" variable is not set. Defaulting to a blank string."
13:33:51 time="2023-11-08T07:33:51+01:00" level=warning msg="The \"PRESERVE_HOST_HEADER_OPTION\" variable is not set. Defaulting to a blank string."
13:33:51 time="2023-11-08T07:33:51+01:00" level=warning msg="The \"USER_UID\" variable is not set. Defaulting to a blank string."
13:33:51 time="2023-11-08T07:33:51+01:00" level=warning msg="The \"USER_GID\" variable is not set. Defaulting to a blank string."
```

https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-record-zuul/106/console